_includes/footer.html

```diff
@@ -2,7 +2,7 @@
 
   <div>
     <div class="primary">
-      <a href="https://okfn.org/" rel="external"><img src="{{ "/img/open-knowledge.png" | prepend: site.baseurl }}" width="145" height="32" alt="Open Knowledge"></a>
+      <a href="https://okfn.org/" rel="external"><img src="{{ "//a.okfn.org/img/oki/landscape-grey-165x43.png" | prepend: site.baseurl }}" height="32" alt="Open Knowledge"></a>
```

Comment:
Did you check this? I think it will not work with the prepend filter still in place...